cmd/operator-sdk/run/local/local.go

```diff
@@ -40,41 +40,42 @@ import (
 	"github.com/operator-framework/operator-sdk/pkg/log/zap"
 )
 
-type runLocalArgs struct {
-	kubeconfig           string
-	watchNamespace       string
-	operatorFlags        string
-	ldFlags              string
-	enableDelve          bool
-	ansibleOperatorFlags *aoflags.AnsibleOperatorFlags
-	helmOperatorFlags    *hoflags.HelmOperatorFlags
+type RunLocalCmd struct {
+	Kubeconfig           string
+	WatchNamespace       string
```

Comment:
```suggestion
        // Valid just for the legacy layout
        // todo: remove when the legacy layout is no longer supported 
	WatchNamespace       string
```

Reply:
Seeing as projects created with `init` can be namespaced (with extra work), I don't see a reason to ever remove this.

Reply:
Furthermore do we want to disable setting `--watch-namespaces`, as we currently do, for new project layouts, considering that they can be namespaced @joelanford @hasbro17? Out of scope for this PR, still worth discussing.

Reply:
Hi @estroz,  It is disabled for kb layout already. I understand that so far we do not think in port this for kb layouts. 
However, may we can plan (create a task) to check if has any reason that would make it valuable be ported.

Reply:
It is currently disabled, and I'm wondering if we want to continue to do that. If a new project layout is made to be namespaced (manager consumes `--namespaces` or `WATCH_NAMESPACE`), shouldn't they be able to use `--watch-namespaces`?

Reply:
The WATCH_NAMESPACE feature requires changes in the scaffold which are not made in the kb layout so was decided that until we have sure that we would like to migrate this feature and it that it would be really valuable the flag would be hidden and not "officially" deprecated. See the comments in the code; https://github.com/operator-framework/operator-sdk/pull/3067/files#diff-3b91b8e7a6ffd108dcc88e06b3c11a01R41-R43

PS: Since it is related to the operator scope setup options as well. I added in the operator scope doc the steps which would be required to have the code changes that also could allow the flags work. See: https://github.com/operator-framework/operator-sdk/pull/2982/files#diff-f0fb41913d0dbfca6bcc82359ab92aacR113-R185

Reply:
We can address this in a follow-up.